<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>484</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Boolean Minimizer</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="truth_table">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>481</width>
      <height>271</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>46</number>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>X1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Y1</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="input_plus">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>230</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>+ input</string>
    </property>
   </widget>
   <widget class="QPushButton" name="term_plus">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>310</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>+ term</string>
    </property>
   </widget>
   <widget class="QPushButton" name="input_minus">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>230</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>- input</string>
    </property>
   </widget>
   <widget class="QPushButton" name="term_minus">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>310</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>- term</string>
    </property>
   </widget>
   <widget class="QPushButton" name="output_plus">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>270</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>+ output</string>
    </property>
   </widget>
   <widget class="QPushButton" name="output_minus">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>270</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>- output</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="output_display">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>691</width>
      <height>111</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
